tools/Azure.Mcp.Tools.Communication/src/Commands/Email/EmailSendCommand.cs

```diff
@@ -38,9 +38,9 @@ Supports HTML content and CC/BCC recipients.
     public override ToolMetadata Metadata => new()
     {
         Destructive = false,
-        ReadOnly = false,
+        ReadOnly = true,
         OpenWorld = true,
-        Idempotent = false,
+        Idempotent = true,
```

Comment:
This doesn't feel correct to me, I don't think we should be modifying `ToolMetadata` so it can be contained in a consolidated group.